nvm.sh

```diff
@@ -3056,6 +3058,7 @@ nvm() {
       local ALIAS
       local NVM_UPGRADE_NPM
       NVM_UPGRADE_NPM=0
+      NVM_WRITE_TO_NVMRC=0
```

Comment:
```suggestion
      local NVM_WRITE_TO_NVMRC
      NVM_WRITE_TO_NVMRC=0
```
this is needed for dash